packages/material/scss/upload/_theme.scss

```diff
@@ -0,0 +1,163 @@
+@include exports('upload/theme') {
+    $upload-text: $panel-text !default;
+    $upload-padding: 14px !default;
+
+    $upload-padding-sm: $popup-padding-y !default;
+
+    $upload-button-bg: rgba(0, 0, 0, 0.12) !default;
+    $upload-button-active-bg: rgba(0, 0, 0, 0.28) !default;
+
+    $upload-border-color: rgba(0, 0, 0, .08) !default;
```

Comment:
@elena-gancheva after the ScrollView branch gets merged, apply the [suggestions for splitting the layout / theme files](https://github.com/telerik/kendo-themes/pull/12/files/8ede27abb3af79c2d9c8138df2ab5ea052553bed#diff-fb4ad1037708b4666473baedd4ef12ff) to this file.